### PR TITLE
feat(k8s): change cluster type MTA-3647

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: containers/kubernetes/api-cli/changing-control-plane-offer.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+In this tutorial, we show you how to change your cluster type using the Scaleway CLI (v2).
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+  - You have generated your [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+  - You have downloaded and installed the most recent version of the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2)
+  - You have [created a cluster](/how-to/create-cluster)
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/api-cli/changing-control-plane-offer.mdx:28]:
```suggestion
  - You have [created a cluster](/containers/kubernetes/how-to/create-cluster)
```

File: containers/kubernetes/api-cli/creating-managing-kubernetes-lifecycle-cliv2.mdx [hunk 2]
```diff
@@ -95,14 +95,23 @@ You can [create](/containers/kubernetes/how-to/create-cluster/) and [manage](/co
 
     `scw k8s cluster create name=name-of-your-cluster`
 
+    <Message type="tip">
+      You can add the `type` argument to specify the cluster type. For an extensive list of the available cluster types, run the [list cluster types CLI command](https://github.com/scaleway/scaleway-cli/blob/master/docs/commands/k8s.md#list-available-cluster-types-for-a-cluster). If you do not specify a cluster type, a cluster of Kapsule type is created by default.<br /><br />
+
+      It is possible to change the cluster type anytime. Depending on your current cluster type, the types you can change to vary. Refer to the [Changing the control plane offer](/changing-control-plane-offer) documentation page for more information.<br /><br />
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/api-cli/creating-managing-kubernetes-lifecycle-cliv2.mdx:101]:
```suggestion
      It is possible to change the cluster type anytime. Depending on your current cluster type, the types you can change to vary. Refer to the [Changing the control plane offer](/containers/kubernetes/api-cli/changing-control-plane-offer) documentation page for more information.<br /><br />
```

File: containers/kubernetes/api-cli/creating-managing-kubernetes-lifecycle-cliv2.mdx [hunk 3]
```diff
@@ -95,14 +95,23 @@ You can [create](/containers/kubernetes/how-to/create-cluster/) and [manage](/co
...
+
+      For more information about cluster types, refer to the [Kubernetes FAQ page](https://www.scaleway.com/en/docs/faq/kubernetes/#can-i-choose-the-type-of-control-plane-for-my-cluster).
```
Comment (ldecarvalho-doc) [inline containers/kubernetes/api-cli/creating-managing-kubernetes-lifecycle-cliv2.mdx:103]:
```suggestion
      For more information about cluster types, refer to the [Kubernetes FAQ page](/faq/kubernetes/#can-i-choose-the-type-of-control-plane-for-my-cluster).
```

File: containers/kubernetes/api-cli/changing-control-plane-offer.mdx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    ClusterTypes.3.Availability     available
+    ClusterTypes.3.MaxNodes         500
+    ClusterTypes.3.CommitmentDelay  30 days
+    ClusterTypes.3.SLA              99.5
+    ClusterTypes.3.Resiliency       high_availability
+    ClusterTypes.3.Memory           16 GB
+    ClusterTypes.3.Dedicated        true
+    TotalCount                      4
+    ```
+2. Retrieve the details of your cluster to determine the commitment period.
```
Comment (RoRoJ) [inline containers/kubernetes/api-cli/changing-control-plane-offer.mdx:95]: Should be step 3

**Review by nerda-codes — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**